### ISSUE TITLE
New Conda Package for UCX

### ISSUE THREAD

**Issue by quasiben** [rapidsai/ucx-py#425]
We need to release a new conda package with IB patches.  This is a somewhat informal release to be used for IB testing.

@pentschev this will be based on UCX master, correct ?

**Comment by pentschev** [comment]
> @pentschev this will be based on UCX master, correct ?

No, we're still basing off of v1.7, but we'll need a fix that is not in master and won't ever be (it's more of a temporary hack that includes CUDA-specific code in the IB codepath), the patch is in https://gist.github.com/pentschev/1a42434b4114fd40f664ef1d7a792bc4 .

Could you update the UCX conda package with the patch above?

**Comment by jakirkham** [comment]
Where was this patch generated from?

**Comment by quasiben** [comment]
This patch was generated from Akshay to help overcome IB registration issues.  Both @pentschev and I have validated the patch works

**Comment by jakirkham** [comment]
Right. Is there a link to the PR, branch, or commit this came from?

**Comment by quasiben** [comment]
There is no PR or branch but rather something short term for others to use while Akshay develops are more holistic solution

**Comment by jakirkham** [comment]
Thanks Ben! 😄
